app/vmalert/web.qtpl

```diff
@@ -30,313 +91,282 @@
+            "unhealthy": `<svg class="d-md-none" width="22" height="22" viewBox="0 0 14 14"><circle cx="7" cy="7" r="6.5" fill="none"/><path stroke-linecap="round" stroke-width="1.5" d="m5 5 4 4M5 9l4-4"/></svg>`,
+            "noMatch": `<svg class="d-md-none" width="22" height="22" viewBox="0 0 116.623 116.623"><path d="M99.114 89.251H57.783l19.181-41.797h22.149a5.971 5.971 0 1 0 0-11.942H82.444L94.591 9.049a5.97 5.97 0 0 0-2.938-7.917 5.97 5.97 0 0 0-7.918 2.936L69.304 35.513H41.662c-2.289-.458-17.685-4.48-17.685-29.542a5.97 5.97 0 1 0-11.942 0c0 18.536 7.157 29.533 14.93 35.514-7.774 5.981-14.93 16.978-14.93 35.514a5.971 5.971 0 1 0 11.942 0c0-25.007 14.886-29.013 17.677-29.542h22.17L44.643 89.253H17.509a5.972 5.972 0 0 0 0 11.943h21.653l-3.196 6.965a5.972 5.972 0 1 0 10.856 4.981l5.479-11.943h46.812a5.975 5.975 0 0 0 .001-11.948"/></svg>`,
+        }
+        currentText := filters[filter]
+	currentIcon := icons[filter]
+    %}
     {%= tpl.Header(r, navItems, "Groups", getLastConfigError()) %}
-        {%code
-            filter := r.URL.Query().Get("filter")
-            rOk := make(map[string]int)
-            rNotOk := make(map[string]int)
-            rNoMatch := make(map[string]int)
-            var groups []apiGroup
-            for _, g := range originGroups {
-                var rules []apiRule
-                for _, r := range g.Rules {
-                    if r.LastError != "" {
-                        rNotOk[g.ID]++
-                    } else {
-                        rOk[g.ID]++
-                    }
-                    if isNoMatch(r) {
-                        rNoMatch[g.ID]++
-                    }
-                    if (filter == "unhealthy" && r.LastError == "") ||
-                       (filter == "noMatch" && !isNoMatch(r)) {
-                        continue
-                    }
-                    rules = append(rules, r)
-                }
-                if len(rules) > 0 {
-                    g.Rules = rules
-                    groups = append(groups, g)
-                }
-            }
-        %}
-        <div class="btn-toolbar mb-3" role="toolbar">
-          <div>
-            <a class="btn {%= buttonActive(filter, "") %}" role="button" onclick="window.location = window.location.pathname">All</a>
-            <a class="btn btn-primary" role="button" onclick="collapseAll()">Collapse All</a>
-            <a class="btn btn-primary" role="button" onclick="expandAll()">Expand All</a>
-            <a class="btn {%= buttonActive(filter, "unhealthy") %}" role="button" onclick="location.href='?filter=unhealthy'" title="Show only rules with errors">Unhealthy</a>
-            <a class="btn {%= buttonActive(filter, "noMatch") %}" role="button" onclick="location.href='?filter=noMatch'" title="Show only rules matching no time series during last evaluation">NoMatch</a>
```

Comment:
Button `unhealthy` and `noMatch` are missed.

Reply:
click on `All` dropdown to select `Unhealthy` or `No Match`

Reply:
I don’t understand why they are all grouped together. No Match and Unhealthy are not conflicting.

Reply:
in a master this filtering is done on a server side and even if these filters are not conflicting only one is applied at the moment. there's a `filter`, which can either be `unhealthy` or `noMatch`

Reply:
I can make these options as checkboxes in a dropdown and it'll make more sense than having three confusing buttons, that are not even close to each other

Reply:
I don't have strong opinion on this. But yes, I find always visible buttons `unhealthy` and `nomatch` just easier to use (i see them right away, i need one click to apply them). Dropdown kinda hinds the option from me, so as a first-timer I won't even know about this filter.
So I'd prefer to have it as explicit buttons, or change the `All` button to `Filter by status` or something like this. So users would understand what it is right away.

Reply:
added `filter by status` dropdown

Reply:
@Haleygo ptal

Reply:
Can we show the current selected status? Currently it's hidden in the dropdown.
<img width="759" alt="image" src="https://github.com/user-attachments/assets/44cb1dea-f2b6-4983-a541-317bc339ad77" />

Reply:
how about this?
<img width="1005" alt="image" src="https://github.com/user-attachments/assets/64668e09-5472-4931-927e-556a67706cba" />